pkg/pb/metadata/metadata.proto

```diff
@@ -67,6 +78,15 @@ message PutFileRequest {
 
 message PutFileResponse { }
 
+message PutFileStreamRequest {
+    // key and metadata should only be set on the first request
+    string key = 1;
+    bytes fileChunk = 2;
+    FileMetadata metadata = 3;
```

Comment:
We have a `SetMetadata` RPC, probably better to use that instead of sending metadata alongside each chunk or implementing an adhoc protocol where metadata is nil for all chunks but the first.

Reply:
I'm fine with this as is too.

Reply:
You're right, doing it this way is a bit janky